virttest/utils_sys.py

```
@@ -74,6 +74,37 @@ def get_host_bridge_id(session=None):
+        LOG.debug(
+            "Detected %s mode by command '%s'",
+            "image" if result else "package",
+            check_command,
+        )
+        return result
+
+    if session:
+        status, output = session.cmd_status_output(check_command)
+        return _check_output(status, output)
+    else:
+        ret = process.run(check_command, shell=True, verbose=True, ignore_status=True)
```

Ruff (0.12.2)

Function call with `shell=True` parameter identified, security issue

(S604)